### PR TITLE
refactor notifications

### PR DESCRIPTION
**Pull request by danielrus**

### CODE REVIEW

File: src/app/about/about.component.ts [hunk 1]
```diff
@@ -8,6 +9,70 @@ import { version } from '../../../package.json';
 })
 export class AboutComponent {
   public version: string = version;
-
   public ngVersion: string = VERSION.full;
+
+  constructor(private notificationService: NotificationService) {}
+
+  notifyInfo(): void {
+    this.notificationService.message('A simple message with usefull informtion.');
```
Comment (sandorbanvitagroup) [inline src/app/about/about.component.ts:17]: typo "useful".

File: src/styles.scss [hunk 2]
```diff
@@ -73,3 +76,54 @@ body {
...
+    }
+
+    .ajs-message {
+      width: auto;
+      max-width: 436px;
+      padding: 2em;
+      margin-bottom: 1em;
+      background-color: map-get($sormas-primary, 50);
+      border-radius: 5px;
+      box-shadow: 0 4px 5px rgba(0, 0, 0, 0.14), 0 1px 10px rgba(0, 0, 0, 0.12),
```
Comment (sandorbanvitagroup) [inline src/styles.scss:106]: Please add a  variable in variables.scss and use it here like : rgba($black, 0.14);

File: src/theme/dialog.scss [hunk 3]
```diff
@@ -0,0 +1,79 @@
...
+    .mat-dialog-title {
+      margin-top: 40px;
+      margin-bottom: 24px;
+      font-size: 24px;
+      line-height: 1;
+    }
+
+    .mat-dialog-content {
+      padding: 16px 64px;
+      border-top: 2px solid #c1cad9;
```
Comment (sandorbanvitagroup) [inline src/theme/dialog.scss:24]: Please use map-get($sormas-accent, 200); instead.

File: src/styles.scss (hunk 2 above)
Comment (danielrus) [reply src/styles.scss]: i've used new way of adding shadows

**Review by sandorbanvitagroup — APPROVED** [review]

**Review by adriandumavitagroup — APPROVED** [review]

**Pull request merged**